src/Sylius/Bundle/AddressingBundle/spec/Form/EventListener/BuildAddressFormSubscriberSpec.php

```diff
@@ -34,12 +36,12 @@ function let(ObjectRepository $countryRepository, FormFactoryInterface $factory)
 
     function it_is_initializable()
     {
-        $this->shouldHaveType('Sylius\Bundle\AddressingBundle\Form\EventListener\BuildAddressFormSubscriber');
+        $this->shouldHaveType(BuildAddressFormSubscriber::class);
```

Comment:
This change should be reverted. In a moment we write spec we can't use `BuildAddressFormSubscriber::class` because this class does not exist yet. What is more, we want to verify the whole namespace together with possible typos. This change makes this spec useless because it will always be true. 

Same thing for `implements` and in every other spec.